sources/commands/option_external.c

```diff
@@ -49,13 +73,14 @@ int	option_external(t_control control, t_treenode *node, t_param *param)
 	*control.found = 1;
```

Comment:
1. if we're at this point, the shell understood that we're trying to run not a builtin command (echo, pwd etc), but an external one